daemon/api_notices.go

```diff
@@ -22,22 +22,28 @@ import (
 	"net/http"
 	"strconv"
 
+	"github.com/snapcore/snapd/dirs"
 	"github.com/snapcore/snapd/overlord/auth"
 	"github.com/snapcore/snapd/overlord/state"
 	"github.com/snapcore/snapd/strutil"
 )
 
+var noticeReadInterfaces = map[state.NoticeType][]string{
+	state.ChangeUpdateNotice:   {"snap-refresh-observe"},
+	state.RefreshInhibitNotice: {"snap-refresh-observe"},
+}
+
 var (
 	noticesCmd = &Command{
 		Path:       "/v2/notices",
 		GET:        getNotices,
-		ReadAccess: openAccess{},
+		ReadAccess: interfaceOpenAccess{Interface: "snap-refresh-observe"},
```

Comment:
We're going to need to add other interfaces to this list here (namely, `snap-prompting-control`), iirc were there plans to make `interfaceOpenAccess` support a list of interfaces?

Reply:
Yes, can confirm, this is something I need for prompting, since `"snap-refresh-observe"` and `"snap-prompting-control"` both need `interfaceOpenAccess` to the notices and snaps endpoints. @ZeyadYasser @pedronis are there plans in the works for this, or should I add this to my queue?

Reply:
We added TODOs already about that here, I don't know whether @ZeyadYasser added it already to his backlog, he mentioned he would, you two should probably coordinate on who can/needs to work on this first.